starport/cmd/network_campaign_publish.go

```diff
@@ -0,0 +1,52 @@
+package starportcmd
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/tendermint/starport/starport/pkg/clispinner"
+)
+
+// NewNetworkCampaignPublish returns a new command to publish a new campaigns on Starport Network.
+func NewNetworkCampaignPublish() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "publish [name] [total-supply]",
```

Comment:
```suggestion
		Use:   "create [name] [total-supply]",
```

I would recommend to rename this to avoid confusion. Otherwise
* chain publish
* campaign publish

names would be too similar.